### ISSUE TITLE
[Feature Request]: Customize Asset Print Page

### ISSUE THREAD

**Issue by Breakingx** [grokability/snipe-it#12895]
### Is your feature request related to a problem? Please describe.

Hello, I'm sorry if a Ticket already exists for this case; I couldn't find any.

It would be pretty helpful if it would be possible to edit the User Assignemnt Print without editing the print.blade.php. We host our application in a read only enviroment. So I'm unable to edit it.

### Describe the solution you'd like

In the settings a section to edit the configuration.

**Comment by snipe** [comment]
I don't really see us implementing this any time soon, if I'm honest. It just leaves things open to people breaking things left and right, and then we end up having to field those "bug" reports for code we didn't write. Can you explain what kind of customizations you'd be wanting?

**Comment by Breakingx** [comment]
Good Point actually. Making this kind of "little" feature opens a door for many problems. 
I just did an edit of the print that would be suitable for _my_ company. Maybe there is a chance that you like it.
[https://i.imgur.com/x3TIuKY.png](url)

For a better overview I rearranged the headlines. 
I talked to some people inside my company and they prefer to see the category at first an than the name of the device because these two informations are the most important one. I combined the AssetName and the AssetTag. The last thing I would like to have changed is that the give out time is just the date. The precision to the second is way to much and confuses the user. "I don't receive my Laptop at 11pm!"

These are my two cents. :)
